src/payment/strategies/clearpay/clearpay-script-loader.ts

```diff
@@ -0,0 +1,38 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+import PaymentMethod from '../../payment-method';
+
+import ClearpaySdk from './clearpay-sdk';
+import ClearpayWindow from './clearpay-window';
+
+const SCRIPTS_DEFAULT = {
+    PROD: '//portal.clearpay.co.uk/afterpay-async.js',
+    SANDBOX: '//portal.sandbox.clearpay.co.uk/afterpay-async.js',
+};
+
+/** Class responsible for loading the Clearpay SDK */
+export default class ClearpayScriptLoader {
+    constructor(
+        private _scriptLoader: ScriptLoader,
+        public _window: ClearpayWindow = window
+    ) { }
+
+    /**
+     * Loads the appropriate Clearpay SDK depending on the payment method data.
```

Comment:
You don't need this docblock, we already have types provided by the SDK.

We only add docblocks for public SDK methods, so devs using the SDK know how to use them. This is an internal method of the SDK, so there's no need to document it.